core/vm/jump_table.go

```diff
@@ -1106,6 +1106,21 @@ func newFrontierInstructionSet() JumpTable {
 			valid:      true,
 			writes:     true,
 		},
+		JUMPSUB: {
+			execute:     opJumpSub,
+			constantGas: GasMidStep,
+			minStack:    minStack(1, 0),
+			maxStack:    maxStack(1, 0),
+			jumps:       true,
+			valid:       true,
+		},
+		RETURNSUB:
+			execute:     opReturnSub,
+			constantGas: params.JumpdestGas,
+			minStack:    minStack(0, 0),
```

Comment:
SO, for this to work,we also need a validation step for the `returns` stack, otherwise the `pop` in `opReturnSub` can panic

Reply:
True.

Reply:
Actually, I think not true.  See below.